nilearn/datasets/tests/test_func.py

```diff
@@ -679,3 +679,17 @@ def test_fetch_development_fmri_exception():
     with pytest.raises(ValueError, match='Wrong value for age_group'):
         func._filter_func_regressors_by_participants(participants='junk',
                                                      age_group='junk for test')
+
+def test_fetch_surf_tva_localizer():
```

Comment:
We must create a better test. Downloading all the files is a long process, and is fragile if the server is interrupted.

Reply:
Any idea? I have to admit I copied the content of the test function on some other existing ones... Isn't this a more global question (more global than what is done in this PR)?

Other than this, I think I fixed everything else...

Reply:
Let me think on this, you can work on BT's comments.